src/CustomSearch.jsx

```diff
@@ -0,0 +1,35 @@
+import React, {useState, useRef, useEffect} from "react";
+
+function CustomSearch() {
+  const [adressForm, setAdressform] = React.useState('')
+  const [adressCoordinates, setAdresscoordinates] = React.useState('') 
+
+  function handleSubmit(event) {
+    event.preventDefault()
+    setAdressform(event.target.suggest.value)
+    
+    var myGeocoder = ymaps.geocode(event.target.suggest.value);
+    myGeocoder.then(
+        function (res) {
```

Comment:
Again. Consider using arrow functions. They're always bound to one context, and it can't be changed, which makes the program behavior simpler and easier to maintain

Reply:
done